site/docs/selection/clear.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+menu: docs
+title: Clearing a selection
+permalink: /docs/clear.html
+---
+
+The `clear` selection transformation resets the visualization to its initial configuration:
+
+- For `single` and `multi` selections, it will clear all selected values.
+- For `interval` selections, it will reset the grid position to its initial configuration.
+
+It can take one of the following values:
+
+- `false` -- disables clear behavior; there will be no trigger that resets the visualization to its initial configuration.
+- A [Vega event stream definition](https://vega.github.io/vega/docs/event-streams/) to indicate which events should trigger clearing of the visualization.
+
+Vega-Lite automatically adds a clear transform to all selections by default. It will check your Vega-Lite configuration and conditionally pick an appropriate `clear` trigger based off of the `on` trigger (e.g. `click` to `dblclick`, `mouseover` to `mouseout`).
```

Comment:
The default behavior should be in the TS Docs (so it shows up in tools like Altair too.) 

> It will check your Vega-Lite configuration and conditionally pick an appropriate `clear` trigger based off of the `on` trigger (e.g. `click` to `dblclick`, `mouseover` to `mouseout`).

This description is a bit unclear what is the default if `on` is not click/mouseover.  (Missing "else" case.)

Reply:
What are the TS Docs?

Reply:
https://github.com/vega/vega-lite/pull/4720/files#diff-a02c9ec474a7c7ff01691b050f063691R60  -- You already added some, but didn't include default behavior. 


https://github.com/Microsoft/tsdoc